### DISCUSSION TITLE
I want to use Mongodb for minetest.

### DISCUSSION THREAD

**Discussion by moondoor01** [community/community#124200] [Discussions]
### Select Topic Area

Question

### Body

Hello.
I am developing a minetest game right now.
But I want to use Mongodb instead of Mysql.
Is there any plugin or something like that for it?
Many thanks.

**Comment by SirBang** [comment]
Hello. I am developing Minetest game too.
But currently, I am using mysql.
I heard there is no plugin for mongodb.
But you can make it on yourself by using C++.
I wish you a good luck.
